pkg/workload/cli/run.go

```diff
@@ -659,3 +670,51 @@ func maybeLogRandomSeed(ctx context.Context, gen workload.Generator) {
 		log.Infof(ctx, "%s", randomSeed.LogMessage())
 	}
 }
+
+func maybeInitAndCreateExporter() (exporter.Exporter, *os.File, error) {
+	var metricsExporter exporter.Exporter
+	var file *os.File
+	if *histograms != "" {
+
+		switch *histogramExportFormat {
+		case "json":
+			metricsExporter = &exporter.HdrJsonExporter{}
+		case "openmetrics":
+			labelValues := strings.Split(*openmetricsLabels, ",")
+			labels := make(map[string]string)
+			for _, label := range labelValues {
+				parts := strings.Split(label, "=")
+				if len(parts) != 2 {
+					return nil, nil, errors.Errorf(`invalid histogram label "%s"`, label)
```

Comment:
Nit: don't really need to invoke special quotes, the `%q` verb is for quoting (string) values,

```
errors.Errorf("invalid histogram label: %q", label)
```